examples/cloudtrail/go/go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/opentracing/opentracing-go v1.1.0 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pkg/term v1.1.0 // indirect
-	github.com/pulumi/pulumi-aws/sdk/v5 v5.35.0 // indirect
+	github.com/pulumi/pulumi-aws/sdk/v5 v5.3.0 // indirect
```

Comment:
This reads like it's lowering the dep? Should it be 6?

Reply:
Not sure why `go mod` downgraded here. The reason it's still on v5 is that it's an indirect dependency coming from the direct import of github.com/pulumi/pulumi-awsx/sdk - catch 22.